### ISSUE TITLE
installer not working

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-pyworkflow#516]
https://github.com/scipion-em/scipion-pyworkflow/blob/2810fede3b1652ee0c7b17d726f8c3778bdcd57d/pyworkflow/config.py#L12

```
Obtaining file:///home/gsharov/soft/xmipp-bundle2/scipion-pyworkflow
Preparing metadata (setup.py) ... error
error: subprocess-exited-with-error

× python setup.py egg_info did not run successfully.
│ exit code: 1
╰─> [10 lines of output]
Traceback (most recent call last):
File "<string>", line 2, in <module>
File "<pip-setuptools-caller>", line 34, in <module>
File "/home/gsharov/soft/xmipp-bundle2/scipion-pyworkflow/setup.py", line 37, in <module>
import pyworkflow as pw
File "/home/gsharov/soft/xmipp-bundle2/scipion-pyworkflow/pyworkflow/__init__.py", line 26, in <module>
from .config import *
File "/home/gsharov/soft/xmipp-bundle2/scipion-pyworkflow/pyworkflow/config.py", line 12, in <module>
from matplotlib.colors import to_rgb
ModuleNotFoundError: No module named 'matplotlib'
[end of output]
```

**Comment by pconesa** [comment]
upss!

**Comment by azazellochg** [comment]
Fixed in #517
